bin/dump-data

```diff
@@ -97,6 +109,7 @@ def generate_dump
       frontend_status,
       backend_status,
       example_url,
+      content_item_coverage
```

Comment:
Sorry one more change, can you move this above the example url field and I'll update the template spreadsheet.